### ISSUE TITLE
ThunderbirdTray exits immediately with no output or activity

### ISSUE THREAD

**Issue by cr1901** [TheGuardianWolf/ThunderbirdTray#7]
I downloaded the most-recent [release](https://github.com/TheGuardianWolf/ThunderbirdTray/releases/download/v1.3.1/ThunderbirdTray.NETFramework.zip), put it into my private binary directory, and attempted to run the application manually just to check if it works. Unfortunately, the application exits immediately without any activity:

```
William@DESKTOP-H0PMN4M C:\Users\William\bin
$ ThunderbirdTray.exe

William@DESKTOP-H0PMN4M C:\Users\William\bin
```

The same behavior occurs when I try to run in debug mode:

```
William@DESKTOP-H0PMN4M C:\Users\William\bin
$ ThunderbirdTray.exe --debug
```

Clicking the icon either from the `Startup` folder or my bin directory in File Explorer has the same results. There was exactly _one_ run tonight where I saw the mail icon in the tray in testing various scenarios of having Thunderbird open or not before starting the tray. Unfortunately, I can't remember the exact conditions to duplicate and think it was a transient success :(.

I wonder if there's something wrong with my environment... is it possible to tell where the application crashed without any output (I don't have the luxury of trying myself right now)?

**Comment by TheGuardianWolf** [comment]
Hello there,

Could you please tell me the following:

* OS Version
* DotNet framework version

In addition, could you try out .NET Core (Self contained) to see if there's any difference?

Is the log.txt file being generated? (Will happen in debug or normal mode)

**Comment by cr1901** [comment]
PowerShell gives me this:

```
PS C:\Users\William> (Get-ItemProperty "HKLM:SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full").Release
528040
```

According to [this page](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed#version_table), that corresponds to .NET 4.8 on Windows 10 (and I'm using Windows 10 Pro :)...).

There is no log.txt to speak of:

```
William@DESKTOP-H0PMN4M C:\Users\William\bin
$ dir
 Volume in drive C has no label.
 Volume Serial Number is 80D3-090D

 Directory of C:\Users\William\bin

01/05/2020  11:43 PM    <DIR>          .
01/05/2020  11:43 PM    <DIR>          ..
01/05/2020  11:43 PM           246,784 ThunderbirdTray.exe
               1 File(s)        246,784 bytes
               2 Dir(s)  276,395,782,144 bytes free
```

**Comment by TheGuardianWolf** [comment]
I noticed that in the dir list, there are no .dll files, where did they go?

Also have you tried the self contained version? if you are just looking for an .exe, I would recommend that one.

**Comment by cr1901** [comment]
@TheGuardianWolf Oh my, I forgot to copy the three .dll files (and for some reason Windows didn't throw an error on missing .dlls). When I copied them, the application works as intended.

>There was exactly one run tonight where I saw the mail icon in the tray in testing various scenarios of having Thunderbird open or not before starting the tray.

Re: this sentence, it turns out Thunderbird _does_ in fact have messages that go into the tray on Windows, but only for new mail (which is different from ThunderbirdTray's purpose). Closing as user error.
